react/features/toolbox/components/native/Toolbox.js

```diff
@@ -44,12 +44,7 @@ type Props = {
     /**
      * Whether or not the reactions feature is enabled.
      */
-    _reactionsEnabled: boolean,
-
-    /**
-     * The redux {@code dispatch} function.
-     */
-    dispatch: Function
```

Comment:
Was this not used? I see you didn't remove any usage of it...

Reply:
It wasn't used at all. No actions being dispatched here.